tests/41end-to-end-keys/08-cross-signing.pl

```diff
@@ -652,6 +652,21 @@
          matrix_join_room( $user2, $room_id );
       })->then( sub {
          sync_until_user_in_device_list( $user1, $user2 );
+      })->then( sub {
+         matrix_get_e2e_keys( $user1, $user2_id );
+      })->then( sub {
+         my ( $content ) = @_;
+
+         log_if_fail "key query content1", $content;
+
+         # Check that we do in fact see the master key when querying the
+         # devices.
+         assert_json_keys( $content->{master_keys}, $user2_id );
+         assert_json_keys( $content->{master_keys}->{$user2_id}, "keys");
+         assert_json_keys( $content->{master_keys}->{$user2_id}{keys},
+            "ed25519:nqOvzeuGWT/sRx3h7+MHoInYj3Uk2LD/unI9kDYcHwk");
```

Comment:
Should we also expect to see this in the second call to matrix_get_e2e_keys on +685?

Reply:
Yes we should. I suppose its worth checking that its still there